package.json

```diff
@@ -52,8 +52,10 @@
     "enzyme": "^3.9.0",
     "enzyme-adapter-react-16": "^1.13.2",
     "jest-localstorage-mock": "^2.4.0",
+    "json-loader": "^0.5.7",
     "react-docgen-typescript-webpack-plugin": "^1.1.0",
     "testcafe": "^1.2.0",
-    "testcafe-react-selectors": "^3.1.0"
+    "testcafe-react-selectors": "^3.1.0",
+    "ts-loader": "^6.0.2"
```

Comment:
Why have we added ts-loader?

Reply:
I think that was an attempt at an approach to loading json, not sure we need it now.

Reply:
Resolved in https://github.com/LBHackney-IT/lbh-etra-front-end/pull/14/commits/b09e38d4f593f367389bfc8dbaedd7dd393c6743.